src/internet_identity/src/assets.rs

```diff
@@ -45,10 +45,7 @@ lazy_static! {
             r#"<script id="setupJs"></script>"#,
             &format!(r#"<script data-canister-id="{canister_id}" id="setupJs">{setup_js}</script>"#).to_string()
         );
-        let index_html = index_html.replace(
-            "CSP_POLICY_PLACEHOLDER",
-            &http::content_security_policy_meta()
-        );
+        let index_html = index_html.replace("<meta replaceme-with-csp/>", &format!(r#"<meta http-equiv="Content-Security-Policy" content="{}" />"#,&http::content_security_policy_meta() ));
```

Comment:
Just noticing now, in HTML you have an extra space

```
<meta replaceme-with-csp />

vs

<meta replaceme-with-csp/>
```

Reply:
not sure it matters?

Reply:
I know, the space is being stripped when Webpack minimizes the HTML, but prettier _really wants_ that space 🙃

Reply:
Oki doki